src/test_l1_dsVideoDevice.c

```diff
@@ -1490,26 +1514,31 @@ void test_l1_dsVideoDevice_positive_dsSetDisplayframerate(void)
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
     // Define variables
     int result;
-    int index = 0;
     intptr_t handle = -1;
     char desiredFramerate[] = "30fps"; // You can modify this value as necessary
 
     // Step 01: Initialize video devices
     result = dsVideoDeviceInit();
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 02: Obtain video device handle
-    result = dsGetVideoDevice(index, &handle);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
-    UT_ASSERT_TRUE(handle >= 0);
-
-    // Step 03: Set the display framerate using the obtained handle
-    result = dsSetDisplayframerate(handle, desiredFramerate);
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    for(int i = 0; i < gDSvideoDevice_NumVideoDevices; i++){
+        result = dsGetVideoDevice(i, &handle);
+        UT_ASSERT_EQUAL(result, dsERR_NONE);
+        UT_ASSERT_TRUE(handle >= 0);
+
+        // Step 03: Set the display framerate using the obtained handle
+        result = dsSetDisplayframerate(handle, desiredFramerate);
```

Comment:
set the supported SupportedDisplayFramerates

Reply:
done